jina/counter.py

```diff
@@ -7,21 +7,21 @@
 
 
 class BaseCounter(Iterator):
-    def __init__(self, seed: int = 0):
+    def __init__(self, seed: int = 0) -> None:
```

Comment:
it's interesting to see monkeytype add `None` to a constructor. normally we never add return type hints to `__init__`, maybe there's a way to disable/ignore it?

Reply:
I will remove myself for the time being yes.

Reply:
I think, the reason `MonkeyType` adds this is - [PEP-0484](https://www.python.org/dev/peps/pep-0484/#the-meaning-of-annotations) says, `__init__` should be annotated with `-> None` [Stackoverflow reference](https://stackoverflow.com/a/46779290/3925392)

Reply:
ah, there's a reason behind it , haha.  Previously we add type hints manually, and we just ignore:
1. Magic methods such as `__enter__`, `__call__`.
2. Private methods started with `_`.
But it's just my personal habbit

Reply:
Oh I just removed them. Should I put them back?

Reply:
I would vote to remove `__init__` . `__enter__` and `__call__` are also good candidates to remove, but at least there is a type.

For "private" methods I vote to add them.

Reply:
take easy, we can have a meeting at some point and clarify everything :)